### PR TITLE
Enable a few more GCC warnings

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

File: src/game/client/components/menus.cpp [hunk 1]
```diff
@@ -1058,8 +1058,6 @@ int CMenus::Render()
 				RenderServerControl(MainView);
 			else if(m_GamePage == PAGE_SETTINGS)
 				RenderSettings(MainView);
-			else if(m_GamePage == PAGE_GHOST)
-				RenderGhost(MainView);
```
Comment (ChillerDragon) [inline src/game/client/components/menus.cpp]: How is this related to gcc ? 😕

Comment (def-) [reply src/game/client/components/menus.cpp]: gcc found this, check a few lines above, the PAGE_GHOST case exists twice ;)

**Comment by heinrich5991** [comment]
Thanks.

bors r+
